src/constants/routeMapper.js

```diff
@@ -16,4 +16,5 @@ export const routes = {
   manageImagesDetailVersion: '/manage-images/:imageId/versions/:imageVersionId',
   repositories: '/repositories',
   learningResources: '/learning-resources',
+  customRepositories: '/preview/insights/content',
```

Comment:
think more clean name 
```JS
customRepositories: '/preview/insights/content',
```
what do you think ?

Reply:
good idea